pom.xml

```
@@ -807,9 +807,56 @@ under the License.</licenseText>
             </dependency>
           </dependencies>
         </plugin>
+        <plugin>
+          <groupId>org.openrewrite.maven</groupId>
+          <artifactId>rewrite-maven-plugin</artifactId>
+          <version>6.8.1</version>
+          <configuration>
+            <activeRecipes>
+            <recipe>org.openrewrite.java.format.AutoFormat</recipe>
+            <recipe>org.openrewrite.java.recipes.JavaRecipeBestPractices</recipe>
+            <recipe>org.openrewrite.java.recipes.RecipeNullabilityBestPractices</recipe>
+            <recipe>org.openrewrite.java.recipes.RecipeTestingBestPractices</recipe>
+            <recipe>org.openrewrite.java.testing.assertj.SimplifyAssertJAssertion</recipe>
+            <recipe>org.openrewrite.java.testing.junit5.JUnit5BestPractices</recipe>
+            <recipe>org.openrewrite.staticanalysis.CodeCleanup</recipe>
+            <recipe>org.openrewrite.staticanalysis.CommonStaticAnalysis</recipe>
+            <recipe>org.openrewrite.staticanalysis.RemoveUnusedLocalVariables</recipe>
+            <recipe>org.openrewrite.staticanalysis.RemoveUnusedPrivateFields</recipe>
+            <recipe>org.openrewrite.staticanalysis.RemoveUnusedPrivateMethods</recipe>
+            </activeRecipes>
+            <exclusions>
+              <exclusion>src/mdo/java/WrapperList.java</exclusion>
+              <exclusion>src/mdo/java/WrapperProperties.java</exclusion>
+            </exclusions>
+            <exportDatatables>true</exportDatatables>
+            <failOnDryRunResults>false</failOnDryRunResults>
```

activate after merge and `run` applied. Its then just one build like in spot to fix offset. @gnodet 
```suggestion
            <failOnDryRunResults>true</failOnDryRunResults>
```